sample/src/main/java/kin/sdk/core/sample/WalletActivity.java

```diff
@@ -30,8 +30,10 @@ public static Intent getIntent(Context context) {
     }
 
     private TextView balance, pendingBalance, publicKey;
+
     private View balanceProgress, pendingBalanceProgress;
-    private DisplayCallback<Balance> balanceCallback, pendingBalanceCallback;
+    private kin.sdk.core.Request<Balance> pendingBalanceRequest;
+    private kin.sdk.core.Request<Balance> balanceRequest;
```

Comment:
can import kin.sdk.core.Request ?

Reply:
I can't because there's already `Request` object here of Volley